sources/europe/fr/FR-IGN-Ortho.geojson

```diff
@@ -0,0 +1,336 @@
+{
+    "type": "Feature",
+    "properties": {
+        "attribution": {
+            "url": "http://openstreetmap.fr/bdortho",
+            "text": "BDOrtho IGN",
+            "required": true
+        },
+        "description": "French IGN aeral imagery",
+        "url": "http://proxy-ign.openstreetmap.fr/bdortho/{zoom}/{x}/{y}.jpg",
```

Comment:
@tyrasd yes it seems that it can be accessed both via `http` and `https`.
They are talking about this on [this openstreetmapFrance thread](http://forum.openstreetmap.fr/viewtopic.php?f=5&t=4715&start=30).